README.rst

```diff
@@ -37,19 +37,19 @@ Copy `config.ini.example` as `config.ini` and edit it with your own credentials.
 Example 1: How do I download the datasets?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We have `plenty of them <https://github.com/datasciencebr/serenata-de-amor/blob/master/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Let's say you want your datasets at `/tmp/serenata-data/`:
+We have `plenty of them <https://github.com/datasciencebr/serenata-de-amor/blob/master/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Let's say you want your datasets at `/data/`:
 
 .. code:: python
 
   from serenata_toolbox.datasets import Datasets
-  datasets = Datasets('/tmp/serenata-data/')
+  datasets = Datasets('data/')
 
   # now lets see what are the latest datasets available
   for dataset in datasets.downloader.LATEST:
       print(dataset)  # and you'll see a long list of datasets!
 
   # and let's download one of them
-  datasets.downloader.download('2016-12-06-reibursements.xz')  # yay, you've just downloaded this dataset to /tmp/serenata-data/
+  datasets.downloader.download('2016-12-06-reibursements.xz')  # yay, you've just downloaded this dataset to /data/
```

Comment:
Actually `/data/` would be the `data` directory in the _root_ of your volume. The correct way would be `data/` (i.e. the data directory inside the current working directory).